projects/packages/forms/src/blocks/step/edit.js

```diff
@@ -0,0 +1,113 @@
+import {
+	useBlockProps,
+	useInnerBlocksProps,
+	store as blockEditorStore,
+} from '@wordpress/block-editor';
+import { useSelect } from '@wordpress/data';
+import { __, sprintf } from '@wordpress/i18n';
+import AttributesControls from './attributes-controls';
+
+import './editor.scss';
+
+// Value used by parent controls to signify showing all steps
+const ALL_STEPS_VALUE = '__all__';
+
+// Define allowed blocks directly in this file to break circular dependency
+const ALLOWED_BLOCKS = [
```

Comment:
If this is the same list as for parent Form block, can this list be shared between them?

Reply:
It is really hard to share this between the two. Without introducing a cicular dependency. 
For now thi sis hard coded but we should implement in a better way once the large refactor lands/